src/rpc/util.h

```diff
@@ -19,6 +19,7 @@
 #include <util/check.h>
 
 #include <string>
+#include <variant>
 #include <vector>
 
 #include <boost/variant.hpp>
```

Comment:
You can drop `#include <boost/variant.hpp>`.

Reply:
Thanks, fixed.